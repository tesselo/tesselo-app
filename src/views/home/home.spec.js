import Vue from 'vue'
import Home from '@/views/home/home'

describe('homepage.vue', () => {
  it('should render correct contents', () => {
    const Constructor = Vue.extend(Home)
    const vm = new Constructor().$mount()
    expect(vm.$el.querySelector('#fetch-employees').textContent)
    .toEqual('Get the Tesselo Squad!!')
  })

  it('should match the snapshot', () => {
    const Constructor = Vue.extend(Home)
    const vm = new Constructor().$mount()

    expect(vm.$el).toMatchSnapshot();
  })
})
