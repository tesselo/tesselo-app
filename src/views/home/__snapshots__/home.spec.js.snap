// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`homepage.vue should match the snapshot 1`] = `
<div
  class="container"
  id="home"
>
  <div>
    <a
      class="tsl-button"
      href="#"
      id="fetch-employees"
      target="_self"
    >
      <div>
        Get the Tesselo Squad!!
      </div>
    </a>
  </div>
   
  <!---->
</div>
`;
